### PR TITLE
[android] Add feedback fab

### LINKED ISSUE #5594: [android] Use a FAB for feedback in the FAQ
**Is your feature request related to a problem? Please describe.**
No.

**Describe the solution you would like**
A bottom bar in the FAQ page with only a `Feedback` option looks weird and out of place. A standard or maybe even extended FAB would look much better and would be easier to grasp/use.

**Describe alternatives you have considered**
None.

**Additional context**
FAB Documentations: 
https://m3.material.io/components/floating-action-button/overview 

https://m3.material.io/components/extended-fab/overview

image

### PR DESCRIPTION
**Pull request by theMr17**
This PR replaces the old Feedback button with a new Feedback floating action button.

Closes #5594

|Before|After|
|--|--|
|image|image|

### CODE REVIEW

**Review by vng — APPROVED** [review]

File: android/app/src/main/java/app/organicmaps/help/FaqFragment.java [hunk 1]
```diff
@@ -63,14 +64,29 @@ protected void doStartActivity(Intent intent)
...
+        if (scrollY > oldScrollY)
+        {
+          feedbackFab.hide();
+        }
+        else
+        {
+          feedbackFab.show();
+        }
+      });
+    }
```
Comment (biodranik) [inline android/app/src/main/java/app/organicmaps/help/FaqFragment.java:88]:
```suggestion
    if (Build.VERSION.SDK_INT >= Build.VERSION_CODES.M)
    {
      root.findViewById(R.id.webview).setOnScrollChangeListener((v, scrollX, scrollY, oldScrollX, oldScrollY) -> {
        if (scrollY > oldScrollY)
          feedbackFab.hide();
        else
          feedbackFab.show();
      });
    }
```

Comment (theMr17) [reply android/app/src/main/java/app/organicmaps/help/FaqFragment.java]: Done

**Comment by Jean-BaptisteC** [comment]
@biodranik WDYT to do same thing with phone number in editor to improve visibility?

**Comment by biodranik** [comment]
@Jean-BaptisteC it's a good idea for consistency and convenience for right-handed users. Some users may not recognize that the "+" FAB icon means "Add phone" though... But it is likely not critical.

There are two other important issues about editing a phone number that should be fixed (I reproduced them on Android 5, maybe newer versions are also affected):

1. When I press Edit Phone, the text field with the phone (it can be empty, or have an existing phone) is not automatically selected, and the numeric/phone virtual keyboard does not appear automatically. Now I need to make another tap on the edit field to start adding the phone. It should be fixed.
2. When I press "Back" or "Apply/Save" in Edit Phone dialog, the virtual numeric keyboard is not hidden and stays on the screen, it's also a bug with the focus than should be fixed.

**Pull request merged**